scriptoocloud/src/main/java/edu/asu/diging/scriptoocloud/config/EmailConfig.java

```diff
@@ -0,0 +1,31 @@
+package edu.asu.diging.scriptoocloud.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.core.env.Environment;
+
+import edu.asu.diging.simpleusers.core.config.SimpleUsers;
+import edu.asu.diging.simpleusers.core.config.SimpleUsersConfiguration;
+
+@Configuration
+@PropertySource("classpath:config.properties")
+public class EmailConfig implements SimpleUsersConfiguration{
```

Comment:
I would call this SimpleUserConfig since it's configuring simple user, not just email.